src/Compilers/VisualBasic/Test/Semantic/Compilation/CompilationAPITests.vb

```diff
@@ -21,6 +21,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
     Public Class CompilationAPITests
         Inherits BasicTestBase
 
+        <WorkItem(8360, "https://github.com/dotnet/roslyn/issues/8360")>
+        <Fact>
+        Public Sub PublicSignWithRelativeKeyPath()
+            Dim options = New VisualBasicCompilationOptions(OutputKind.DynamicallyLinkedLibrary).
+                    WithPublicSign(True).WithCryptoKeyFile("test.snk")
+            AssertTheseDiagnostics(VisualBasicCompilation.Create("test", options:=options),
+<errors>
+BC2014: the value 'test.snk' is invalid for option 'CryptoKeyFile'
+BC37254: Public sign was specified and requires a public key, but no public key was specified
```

Comment:
Isn't this error misleading? It looks like a cascading error. If that is the case, let's try to suppress it.